### ISSUE TITLE
The master and insider branches run the "./build/build. sh" command, but do not build an installation version, only a non installation version. Why is this?

### ISSUE THREAD

**Issue by kevinKaikai** [VSCodium/vscodium#1785]
![image](https://github.com/VSCodium/vscodium/assets/37619768/3d2904c6-f3f8-4f72-ad06-618fea6d2a82)

![image](https://github.com/VSCodium/vscodium/assets/37619768/8243c409-194e-4e6e-9393-719e7e08780b)

**Perhaps it is because the file is missing in subsequent versions**
![image](https://github.com/VSCodium/vscodium/assets/37619768/91e718a2-349d-4dbf-88c1-8a7fb67f2f4a)

But now I don't know the reason why the content is missing after compiling due to the lack of this file

**Comment by daiyam** [comment]
Use `./build/build.sh -p` to build the installers

**Comment by kevinKaikai** [comment]
> Use `./build/build.sh -p` to build the installers



> Use `./build/build.sh -p` to build the installers
thanks，I will try tomorrow

**Comment by kevinKaikai** [comment]
![image](https://github.com/VSCodium/vscodium/assets/37619768/ff8968b4-67af-4191-883c-97b8c632d2bc)
![image](https://github.com/VSCodium/vscodium/assets/37619768/0c948279-604a-433d-a20a-72407a29f38a)
![image](https://github.com/VSCodium/vscodium/assets/37619768/dc36d687-a551-43e2-9a26-9f76fe69d8a1)

Use `./build/build.sh -p `

There is only one compressed package and no installed version. How can I solve this problem?

**Comment by daiyam** [comment]
I think you are missing the [WiX Toolset](http://wixtoolset.org/releases/) which is used to create the `.msi`
What error do you have?

**Comment by kevinKaikai** [comment]
Firstly, thank you very much.
I had found what I need.
![2ec318cef3acd623c86ca1eeefb91e6](https://github.com/VSCodium/vscodium/assets/37619768/02a07f74-a197-4ebc-945b-7b26f6c450a4)

But I encountered a new problem.
I encountered an error while compiling using vscodium 1.84.2.23319
![image](https://github.com/VSCodium/vscodium/assets/37619768/0bd61270-6067-4d28-b97c-9f14a17a9334)

Do I need to change the version? Or is there a corresponding solution?
